CRM/Civigiftaid/Upgrader.php

```diff
@@ -26,6 +26,14 @@ class CRM_Civigiftaid_Upgrader extends CRM_Civigiftaid_Upgrader_Base {
    * Example: Run an external SQL script when the module is installed
    */
   public function install() {
+    // For some reason, possibly to do with the way auto_install.xml is handled
+    // and the fact that it includes `<option_group_id>1</>`, we end up with
+    // the underlying table having a default batch value of '1' - it should be
+    // NULL. Note that this query (or similar) is included in an old upgrade
+    // (sql/upgrade_20.sql), so it's likely that only sites installing this new
+    // suffer.
+    CRM_Core_DAO::executeQuery("ALTER TABLE civicrm_value_gift_aid_submission MODIFY `batch_name` VARCHAR(255) NULL DEFAULT NULL");
```

Comment:
How about putting this in it's own upgrader function, removing the sql file and calling it from here + `migrateOneToTwo()`?